core/msd/src/main/rdl/Workload.tdl

```diff
@@ -87,4 +87,15 @@ type BulkWorkloadRequest Struct {
 type BulkWorkloadResponse Struct {
     Array<DomainServices> unmodifiedServices;   // list of services grouped by domain, those are not changed since time stamp in matchingTag
     Workloads workloads;                        // matching workloads
+}
+
+// generic instance
+type CompositeInstance Struct {
+    DomainName domainName;                  // name of the domain
+    EntityName serviceName;                 // name of the service
```

Comment:
if this only refers to the actual service name (e.g. api) and not the full service name (e.g. athenz.api) since we have a domainName field, then it must be defined as SimpleName.

Reply:
It will have only the service name without domain name, shouldn't it be inconsistent with other endpoints 
For example
Static Workload https://github.com/AthenZ/athenz/blob/master/core/msd/src/main/rdl/Workload.rdli#L75
Dynamic Workload  https://github.com/AthenZ/athenz/blob/master/core/msd/src/main/rdl/Workload.rdli#L43
both accepts service name with out domain name as prefix